tests/tests_restore_dirs.yml

```
@@ -0,0 +1,54 @@
+---
+- name: Check if selinux role restores directory SELinux context
+  hosts: all
+  tasks:
+    - name: Create tempdir, set fcontext and relabel
+      block:
+        - name: Create temporary directory
+          ansible.builtin.tempfile:
+            state: directory
+            prefix: linux_system_role.selinux
+          register: tempdir
+
+        - name: Create temporary file
+          ansible.builtin.tempfile:
+            path: "{{ tempdir.path }}"
+            state: file
+            suffix: temp
+          register: tempfile
+
+        - name: Set default_t context to {{ tempdir.path }}
+          ansible.builtin.command: chcon -R -t default_t "{{ tempdir.path }}"
```

```suggestion
          command: chcon -R -t default_t {{ tempdir.path | quote }}
```
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#manipulating-strings

perhaps there are other cases in the selinux role which use `command: ... "{{ some_variable }}"` but for new code it is preferred to use `{{ some_variable | quote }}` instead.

---

I guess the quotes are not necessary at all given that it's directory generated by `mktemp`

---

fixed